pkg/bootstrap/docker/openshift/helper.go

```diff
@@ -243,7 +244,7 @@ func (h *Helper) Start(opt *StartOptions, out io.Writer) (string, error) {
 	skipCreateConfig := false
 	if opt.UseExistingConfig {
 		var err error
-		configDir, err = h.copyConfig(opt.HostConfigDir)
+		configDir, err = h.copyConfig()
```

Comment:
why are we assuming a hardcoded config path instead of using the user provided config path?